api/src/paths/dwc/dataset/{submissionId}/validate.ts

```diff
@@ -0,0 +1,114 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { PROJECT_ROLE } from '../../../../constants/roles';
+import { getDBConnection } from '../../../../database/db';
+import { defaultErrorResponses } from '../../../../openapi/schemas/http-responses';
+import { authorizeRequestHandler } from '../../../../request-handlers/security/authorization';
+import { DarwinCoreService } from '../../../../services/dwc-service';
+import { getLogger } from '../../../../utils/logger';
+
+const defaultLog = getLogger('paths/dwc/dataset/{submissionId}/validate');
+
+export const POST: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          validSystemRoles: [PROJECT_ROLE.PROJECT_LEAD, PROJECT_ROLE.PROJECT_EDITOR],
```

Comment:
Can delete this line.

The discriminator type `SystemUser` only checks that a user is a member of the system, but makes no role checks. 

Ideally this would get flagged as incorrect, but there isn't a great way to enforce this right now using types/interfaces unfortunately. That being said, `validSystemRoles` won't break anything, it will just be unused, but still best to remove it.